client-java/instrumentation/src/main/java/org/evomaster/client/java/instrumentation/RedisCommand.java

```diff
@@ -0,0 +1,104 @@
+package org.evomaster.client.java.instrumentation;
+
+import java.io.Serializable;
+
+/**
+ * Info related to Redis commands execution.
+ */
+public class RedisCommand implements Serializable {
+    /**
+     * Redis commands we'd like to capture. Extendable to any other command in Redis that may
+     */
+    public enum RedisCommandType {
+        GET,
+        HGET,
+        HGETALL,
+        SMEMBERS,
+        ZRANGE
+    }
+
+    /**
+     * Enumerated type of command.
+     */
+    private final RedisCommandType type;
+
+    /**
+     * Key used in query
+     */
+    private final String key;
+
+    /**
+     * Additional parameter used in some commands like HGET.
+     */
+    private final String subKey;
+
+    /**
+     * Expected value before query
+     */
+    private final Object expectedValue;
```

Comment:
I am not sure if I understand what it means the ``expectedValue'' before an actual query was executed